Documentation/installation/osx/install.md

```diff
@@ -16,15 +12,15 @@ Now you can install delve using `go get`:
 $ go get -u github.com/derekparker/delve/cmd/dlv
 ```
 
-With this method you will not be able to use delve's native backend.
+With this method you will not be able to use delve's native backend, *but you don't need it anyway*.
```

Comment:
We should mention that the native backend is unsupported and no longer really maintained.